etl/steps/data/garden/demography/2023-03-31/population/__init__.py

```diff
@@ -0,0 +1,289 @@
+"""OMM population dataset.
+
+This dataset contains population data from various sources. It also includes former countries.
+
+Four sources are used overall:
+
+- After 1950:
+    - UN WPP (2022)
+- Before 1950:
+    - Gapminder (v7): This is prioritised over HYDE.
+    - HYDE (v3.2)
+    - Gapminder (Systema Globalis):
+        Provides data on former countries, and complements other sources with data on missing years for some countries.
+        More on this dataset please refer to module gapminder_sg.
+"""
+import os
+from copy import deepcopy
+from typing import List, cast
+
+import pandas as pd
+from owid.catalog import Dataset, Table
+from owid.catalog.utils import underscore_table
+from structlog import get_logger
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder
+
+from .gapminder import load_gapminder
+from .gapminder_sg import (
+    load_gapminder_sys_glob_complement,
+    load_gapminder_sys_glob_former,
+)
+from .hyde import load_hyde
+from .unwpp import load_unwpp
+
+log = get_logger()
+
+# naming conventions
+N = PathFinder(__file__)
+METADATA_PATH = os.path.join(N.directory, "meta.yml")
+
+# sources names
+# this dictionary maps source short names to complete source names
+SOURCES_NAMES = {
+    "unwpp": "United Nations, World Population Prospects 2022 (https://population.un.org/wpp/Download/Standard/Population/)",
+    "gapminder_v6": "Gapminder v6 (https://www.gapminder.org/data/documentation/gd003/)",
```

Comment:
For consistency, you could rename this key to simply `gapminder`, and then change all instances of `gapminder_v6` in the file.